src/annotator/notebook.js

```diff
@@ -0,0 +1,45 @@
+import Delegator from './delegator';
+
+export default class Notebook extends Delegator {
+  constructor(element, config) {
+    super(element, config);
+    this.config = config;
```

Comment:
The `Delegator` constructor will already initialize `this.options` to the value of `this.config`, although I'm thinking we should consider removing that functionality from the base class since not all components use it. The shared event bus is the main functionality that subclasses rely on.

Reply:
Right! I forgot about that.